sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/AmqpRetryPolicy.java

```diff
@@ -69,17 +69,15 @@ public int getMaxRetries() {
      * is no longer eligible to be retried.
      */
     public Duration calculateRetryDelay(Throwable lastException, int retryCount) {
-        if (retryOptions.getDelay() == Duration.ZERO
-            || retryOptions.getMaxDelay() == Duration.ZERO
+        if (retryOptions.getDelay() == Duration.ZERO || retryOptions.getMaxDelay() == Duration.ZERO
             || retryCount > retryOptions.getMaxRetries()) {
             return null;
         }
 
         final Duration baseDelay;
         if (lastException instanceof AmqpException && isRetriableException(lastException)) {
             baseDelay = ((AmqpException) lastException).getErrorCondition() == SERVER_BUSY_ERROR
-                ? retryOptions.getDelay().plus(SERVER_BUSY_WAIT_TIME)
-                : retryOptions.getDelay();
+                ? retryOptions.getDelay().plus(SERVER_BUSY_WAIT_TIME) : retryOptions.getDelay();
```

Comment:
For tertiary conditions, I think we should make the `if` and `else` case be in separate lines to be consistent and easy to ready.